core-test/src/test/scala/org/bitcoins/core/protocol/ln/currency/LnCurrencyUnitSpec.scala

```diff
@@ -58,11 +58,6 @@ class LnCurrencyUnitSpec extends Properties("LnCurrencyUnitSpec") {
         else Try(num1 * num2).isFailure
     }
 
-  property("Convert negative LnCurrencyUnit value to Satoshis") =
```

Comment:
removed this because it doesn't really make sense to have a negative ln currency unit